include/experimental/fundamental/v3/std/std_tuple.hpp

```diff
@@ -61,6 +61,10 @@ inline namespace fundamental_v3
 #endif
 #if __cplusplus >= 201402L || defined JASEL_DOXYGEN_INVOKED
 
+namespace functor {
+  template <>
+  struct traits<tuple<_t>> : product_type::as_functor {};
```

Comment:
I need to think a little bit about this one, but I believe it is the same than for std::pair. 
The issue here is that the associated make function takes a variadic number of parameters. 

Please, extract this to another issue so that we can manage it separately, independently of the warnings